Source/WebCore/platform/mock/MockRealtimeMediaSourceCenter.cpp

```diff
@@ -76,7 +76,8 @@ static inline Vector<MockMediaDevice> defaultDevices()
                     { { 112, 112 },  { { 30, 30}, { 27.5, 27.5}, { 25, 25}, { 22.5, 22.5}, { 20, 20}, { 17.5, 17.5}, { 15, 15}, { 12.5, 12.5}, { 10, 10}, { 7.5, 7.5}, { 5, 5} }, 1, 1 },
                 },
                 Color::black,
-                { },
+                { }, // whiteBalanceModes
+                false, // supportsTorch
```

Comment:
It would be good to add a test that uses that camera and check torch.

Reply:
Will do